lib/purchases_flutter.dart

```diff
@@ -956,35 +983,40 @@ class UpgradeInfo {
   ProrationMode? prorationMode;
 
   /// Constructs an UpgradeInfo
+  @Deprecated('Use ProductChangeInfo')
   UpgradeInfo(this.oldSKU, {this.prorationMode});
 }
 
 /// Replace SKU's ProrationMode.
 enum ProrationMode {
   /// The Upgrade or Downgrade policy is unknown.
+  @Deprecated('Use a supported ProrationMode')
   unknownSubscriptionUpgradeDowngradePolicy,
 
   /// Replacement takes effect immediately, and the remaining time will be
   /// prorated and credited to the user. This is the current default behavior.
   immediateWithTimeProration,
 
+  // LATER: Reenable when supported by Android V6
   /// Replacement takes effect immediately, and the billing cycle remains the
   /// same. The price for the remaining period will be charged. This option is
   /// only available for subscription upgrade.
-  immediateAndChargeProratedPrice,
+  // immediateAndChargeProratedPrice,
 
   /// Replacement takes effect immediately, and the new price will be charged on
   /// next recurrence time. The billing cycle stays the same.
   immediateWithoutProration,
 
+  // LATER: Reenable when supported by Android V6
   /// Replacement takes effect when the old plan expires, and the new price will
   /// be charged at the same time.
-  deferred,
+  // deferred,
 
+  // LATER: Reenable when supported by Android V6
   /// Replacement takes effect immediately, and the user is charged full price
   /// of new plan and is given a full billing cycle of subscription,
   /// plus remaining prorated time from the old plan.
-  immediateAndChargeFullPrice
+  // immediateAndChargeFullPrice
```

Comment:
What's going to happen with these commented out ones?

Reply:
These will get commented back in when we support these modes in V6 (which should be soon ™) 😊